tests/conftest.py

```diff
@@ -5,6 +5,10 @@
 from nextcloud_mcp_server.client import NextcloudClient
 from httpx import HTTPStatusError
 import asyncio
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
```

Comment:
I believe this is the reason that tests are currently failing - please remove load_dotenv from this PR. I'd prefer modifying config management in a separate PR.